browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -143,6 +143,42 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
   </translations>
   <release seq="1" allow_pseudo="false">
     <messages fallback_to_english="true">
+      <message name="IDS_BRAVE_LIKING"  desc="Text for Liking Brave">
+        Liking Brave?
+      </message>
+      <message name="IDS_TELL_US_WHAT_YOU_THINK"  desc="Text for Tell us what you think!">
+        Tell us what you think!
+      </message>
+      <message name="IDS_RATE_BRAVE"  desc="Text for Rate Brave">
+        Rate Brave
+      </message>
+      <message name="IDS_BRAVE_RATING_THANKS"  desc="Text for Brave Rating Thanks">
+        Thanks!
+      </message>
+      <message name="IDS_ADD_YOUR_SUGGESTIONS"  desc="Text for Add your suggestions">
+        Add your suggestions
+      </message>
+      <message name="IDS_BRAVE_OPTIONAL"  desc="Text for Optional">
+        Optional
+      </message>
+      <message name="IDS_IN_ONE_EMOJI_EXPERIENCE_WITH_BRAVE"  desc="Text for In one emoji, how's your experience with Brave?">
+        In one emoji, how's your experience with Brave?
+      </message>
+      <message name="IDS_RATE_NOW"  desc="Text for Rate now">
+        Rate now
+      </message>
+      <message name="IDS_LIKING_BRAVE"  desc="Text for Liking Brave?">
+        Liking Brave?
```

Comment:
Is this string a duplicate of IDS_BRAVE_LIKING?

Reply:
updated